pkg/report/testdata/freebsd/report/5

```diff
@@ -0,0 +1,39 @@
+TITLE: Fatal trap 9: general protection fault while in kernel mode in unp_dispose
```

Comment:
It make sense to shorten this a bit.
The log will contain output as is, but these titles become email subjects, show up on dashboard, etc. Some email clients split lines at 80 cols and so syz dup commands become problematic.
At least "while in kernel mode" looks excessive for kernel crashes.

Reply:
I got rid of "while in kernel mode".